.github/workflows/release.yml

```diff
@@ -22,11 +22,11 @@ jobs:
           - php-version: '8.0'
             alpine-version: '3.16'
           - php-version: '8.1'
-            alpine-version: '3.18'
+            alpine-version: '3.21'
           - php-version: '8.2'
-            alpine-version: '3.18'
+            alpine-version: '3.21'
           - php-version: '8.3'
-            alpine-version: '3.18'
+            alpine-version: '3.21'
```

Comment:
any idea how to avoid declaring the list in 2 places?
as we have 2 places, they diverge over time ;( 

https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/master/compose.yaml#L28

Reply:
Yes it could be avoidable, but will involve quite big changes or refactoring on Dockerfile, compose files and some workflows. I could work on a new feature branch as a PoC.